ambry-cloud/src/main/java/com.github.ambry.cloud/CloudMessageReadSet.java

```diff
@@ -0,0 +1,135 @@
+/**
+ * Copyright 2019 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.cloud;
+
+import com.github.ambry.commons.BlobId;
+import com.github.ambry.store.MessageReadSet;
+import com.github.ambry.store.StoreKey;
+import com.github.ambry.utils.ByteBufferOutputStream;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.channels.WritableByteChannel;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+class BlobReadInfo {
```

Comment:
Can this be static inner class?
Also javadoc

Reply:
sure.

Reply:
sure